docs/tutorial/part-four/index.md

```diff
@@ -200,27 +200,27 @@ module.exports = {
 }
 ```
 
-Add the above files and then run `gatsby develop`, per usual, and you should see the following:
+Füge die oberen Dateien hinzu und führe danach wie immer `gatsby develop` aus, du solltest nun folgendes sehen:
 
 ![start](start.png)
 
-You have another small site with a layout and two pages.
+Du hast eine weitere kleine Website mit einem Layout und zwei Seiten.
```

Comment:
Not sure if this would be correct in this context. AFAIK a site would mean a website which may be container of one or multiple pages. Therefore "Webseite" being a "(web)page".